modules/qualitycontrol/ReceiveVideoController.js

```diff
@@ -7,6 +7,25 @@ import { MediaType } from '../../service/RTC/MediaType';
 const logger = getLogger(__filename);
 const MAX_HEIGHT = 2160;
 const LASTN_UNLIMITED = -1;
+const ASSUMED_BANDWIDTH_BPS = {
+    MAX_VALUE: 10000000,
+    MIN_VALUE: -1
+};
+
+/**
+ * Sanitizes the assumed bandwidth bps value to be numeric and within its interval.
+ * @param {number} value - the count of bps.
+ * @returns {number} the sanitized assumed bandwidth bps value.
+ */
+const sanitizeAssumedBandwidthBps = value => {
+    if (isNaN(value) || value > ASSUMED_BANDWIDTH_BPS.MAX_VALUE) {
+        return ASSUMED_BANDWIDTH_BPS.MIN_VALUE;
```

Comment:
I guess you meant the MAX_VALUE here?

Reply:
No no, the negative values are ignored, so it's intended as a reset mechanism, to ignore this setting.